### PR TITLE
Added animation and transition effects to faq & added the user button

### LINKED ISSUE #94: Add Clickable Profile Menu with Dropdown Options
Description: Enhance the profile icon functionality by making it clickable and providing a dropdown menu for profile-related actions. This will improve user experience and provide quick access to profile-related features. The dropdown menu should appear when the user clicks on the Profile icon in the navbar.

Tasks:

Clickable Profile Icon:

Make the profile icon clickable, triggering a dropdown menu.
Ensure the menu opens smoothly (with a slight animation or transition).
Profile Dropdown Menu:

Include the following options in the dropdown:
View Profile
Edit Profile
Account Settings
Logout
Each option should be clearly labeled and spaced appropriately.
Responsiveness:

Ensure the profile dropdown menu is fully responsive and functions well on both desktop and mobile devices.
On smaller screens, consider the placement of the dropdown and make sure it does not interfere with other elements.
Highlight Active Option (Optional):

Optionally, highlight the currently active option within the profile menu (e.g., if the user is on the "Edit Profile" page, that option should be highlighted in the dropdown).
Logout Functionality:

Ensure the Logout button triggers the appropriate logout functionality and clears user sessions/tokens as required.
Accessibility:

Ensure the dropdown menu is accessible via keyboard navigation and includes appropriate ARIA labels for screen readers.

@himeshparashar please assign it to me under gssoc-ext and hacktoberfest

### LINKED ISSUE #95: Add Animations to FAQ Section
Description:
Enhance the user experience by adding smooth animations to the FAQ section. The animations should improve the interactivity and visual appeal when a user expands or collapses a question.

Requirements:

Expand/Collapse Animation:
Apply a smooth sliding animation when expanding and collapsing the FAQ items.
The transition should be fast enough to keep the interaction responsive but slow enough for a smooth effect (e.g., 0.3s - 0.5s duration).

Icon Animation:
Rotate or change the icon next to each question when expanded and revert back when collapsed.
The icon animation should synchronize with the expand/collapse animation.

Fade-in Effect:
Add a subtle fade-in effect for the content when a question is expanded.
Responsive Behavior:

Ensure that the animations work well across all device sizes (desktop, tablet, mobile).

Acceptance Criteria:
 The FAQ section should animate smoothly when a question is expanded or collapsed.
 Icons should transition visually to indicate the expanded/collapsed state.
 Animations should be consistent across all supported browsers and devices.
 No performance issues should arise due to the added animations.

@himeshparashar please assign it to me under gssoc-ext and hacktoberfest

### PR DESCRIPTION
**Pull request by sezallagwal**
closes #95 
closes #94 

https://github.com/user-attachments/assets/8ef5f192-d94f-4c2d-a7e5-414bf5ae2fae

https://github.com/user-attachments/assets/1f090121-55c8-44fc-84e8-ef13f6792791

### CODE REVIEW

File: app/notifications/page.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default function Notifications() {
```
Comment (sudhanshutech) [inline app/notifications/page.tsx:1]: if the pr is to add for animation effects then what this change for ?

File: app/help/page.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export default function help() {
```
Comment (sudhanshutech) [inline app/help/page.tsx:1]: and this too?

**Comment by sezallagwal** [comment]
@sudhanshutech fixed it, you can check

**Comment by sezallagwal** [comment]
@himeshparashar hey can you add gssoc-ext label also

**Pull request merged**